src/cart/SConscript

```diff
@@ -22,6 +22,18 @@ SRC = ['crt_bulk.c', 'crt_context.c', 'crt_corpc.c',
        'crt_swim.c', 'crt_tree.c', 'crt_tree_flat.c', 'crt_tree_kary.c',
        'crt_tree_knomial.c', 'crt_hlc.c', 'crt_hlct.c']
 
+LIB_SRC = ['crt_utils.c']
+
+####################################
+# Create cart utility share objects
+####################################
+def build_utility_shared_obj(denv):
+    """Build cart utility library"""
+    Import('prereqs')
```

Comment:
This is imported already in the calling function.  Just pass as an argument

Reply:
548aaee0bfe5a64845f8cf6e8b6ea3b9c7f58c62